tests/acceptance/blocks/universalresultscomponent.js

```diff
@@ -23,4 +28,42 @@ export default class UniversalResultsComponentBlock {
 
     return sections;
   }
+
+  /**
+   * Wait for results to load on page by checking query response status and searchComplete state
+   * (timeout is set to 10 seconds)
+   */
+  async waitOnSearchComplete () {
```

Comment:
What do you think about making a ResultsLogger wrapper that would handle waiting for a request? Feels a little weird to sandwich it with the component classes

Reply:
I agree, I think we should isolate this request logger code so that we don't have to duplicate it in both the unviversalresultscomponent and in the verticalresultscomponent

Reply:
yup, will do!